hello/lib/hello_web/router.ex

```diff
@@ -8,6 +8,7 @@ defmodule HelloWeb.Router do
     plug :put_root_layout, {HelloWeb.LayoutView, :root}
     plug :protect_from_forgery
     plug :put_secure_browser_headers
+    plug HelloWeb.Plugs.Locale, "en"
   end
 
   pipeline :api do
```

Comment:
@Vretamoza what a pipeline is?

Reply:
A pipeline is a set of plugs which are introduced in a scope in order to pass all the requests in that scope through all of those plugs.